pkgs/applications/misc/phoc/default.nix

```diff
@@ -0,0 +1,68 @@
+{ stdenv
+, fetchFromGitLab
+, meson
+, ninja
+, pkgconfig
+, wrapGAppsHook
+, libinput
+, gnome3
+, glib
+, gtk3
+, wayland
+, dbus
+, cmake
+, libdrm
+, libxkbcommon
+, wlroots
+}:
+
+stdenv.mkDerivation rec {
+  pname = "phoc";
+  version = "0.1.7";
+
+  src = fetchFromGitLab {
+    domain = "source.puri.sm";
+    owner = "Librem5";
+    repo = pname;
+    rev = "3d68b091d0075bd549f6fbb84d6c152f01a2354e";
```

Comment:
It is preferred to use a tag `rev = "v${version}"` when available. It will make updating easier in the future.

Reply:
done!